dops/src/interface/role.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { Role } from '../enum/roles.enum';
+
+export interface IRole {
```

Comment:
You may want to add documentation for how one should use this configuration and what takes precedence.

Reply:
discussed on a call and @praju-aot  will add it in a followup PR